scanner/src/funTest/kotlin/storages/AbstractStorageFunTest.kt

```diff
@@ -155,7 +154,9 @@ abstract class AbstractStorageFunTest(vararg listeners: TestListener) : WordSpec
 
             "not store a result for the same scanner and provenance twice" {
                 val summary1 = scanSummaryWithFiles
-                val summary2 = scanSummaryWithFiles.copy(packageVerificationCode = "anotherPackageVerificationCode")
+                val summary2 = scanSummaryWithFiles.copy(
+                    startTime = scanSummaryWithFiles.startTime.plusSeconds(10)
```

Comment:
Unrelated change?

Reply:
This change was intended. IIUC the removed code intended to create a second summary which is different from the first one. In particular it was only targeting difference in general - not difference in terms of package verification code. This is why I choose to achive the (assumed) goal by modifying the timestamp instead.

Reply:
I see. This wasn't obvious to me, but it makes sense.

Reply:
yeah, it would have been better I mentioned it in the commit.